README.md

```diff
@@ -163,6 +163,10 @@ Disable xwayland with `meson -Dxwayland=disabled build/`
 
 For OS/distribution specific details see see [wiki].
 
+To use the supplied wlroots.wrap file, run:
+
+    meson setup build --force-fallback-for=wlroots
```

Comment:
Suggest `s/use/enforce/` or something like that to get the point across that this will ignore whatever wlroots is installed on the system.

Maybe we should also mention using the `meson install --skip-subprojects` flag if installing after using the `wlroots.wrap` file to prevent installing the wlroots headers.

Ref
- #752

Reply:
Thanks.
Both done.